.gitlab-ci.yml

```diff
@@ -42,6 +42,8 @@ build:
     - |
       docker build \
         --build-arg "NPM_REGISTRY=https://nexus.osdc.io/repository/npm-all/" \
+        --build-arg BASE_CONTAINER_REGISTRY \
+        --build-arg BASE_CONTAINER_VERSION \
```

Comment:
Can we piggyback the missing docker labels?
git-commit, image source (repo), etc.?

Reply:
updated:
```
            "Labels": {
                "org.opencontainers.image.created": "2024-02-16T22:12:40+00:00",
                "org.opencontainers.image.description": "portal-ui container image",
                "org.opencontainers.image.documentation": "https://aws.amazon.com/linux/amazon-linux-2023/faqs/",
                "org.opencontainers.image.ref.name": "nginx-extras:3.0.0",
                "org.opencontainers.image.revision": "7295644975f06f14b4be6e7f58cc09ffa837618d",
                "org.opencontainers.image.source": "https://github.com/NCI-GDC/portal-ui",
                "org.opencontainers.image.title": "portal-ui",
                "org.opencontainers.image.vendor": "NCI GDC",
                "org.opencontainers.image.version": "3.0.0",
                "org.opencontainers.ref.name": "portal-ui:feat_dev-2511-base-container-v3"
            }
```

Reply:
awg:

```
            "Labels": {
                "org.opencontainers.image.created": "2024-02-16T22:25:57+00:00",
                "org.opencontainers.image.description": "portal-ui-awg container image",
                "org.opencontainers.image.documentation": "https://aws.amazon.com/linux/amazon-linux-2023/faqs/",
                "org.opencontainers.image.ref.name": "portal-ui-awg:feat_dev-2511-base-container-v3",
                "org.opencontainers.image.revision": "66d379faf531b5742aa0aeeca387452b563c6209",
                "org.opencontainers.image.source": "https://github.com/NCI-GDC/portal-ui",
                "org.opencontainers.image.title": "portal-ui-awg",
                "org.opencontainers.image.vendor": "NCI GDC",
                "org.opencontainers.image.version": "3.0.0"
            }
```